src/microbe_stage/BackgroundPlane.cs

```diff
@@ -166,9 +171,8 @@ protected override void Dispose(bool disposing)
         base.Dispose(disposing);
     }
 
-    private void UpdateSubViewportResolution()
+    private void UpdateSubViewportResolution(Vector2I newSize)
     {
-        var newSize = GetWindow().Size;
```

Comment:
I think it would be clearer to put the logic here to determine which size to use. That way accidentally calling UpdateSubViewportResolution from the wrong place in the code would still work correctly. And would only require `GetWindow().Size` to be in one place which I think is a relatively "complex" operation. 

Also I think in _Ready if low quality blur is on already it should use the fixed resolution, so this refactoring I think would fix a bug as well.